apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/AnalyticsTest.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
This test must use analytics listener and check event is sent with the right values, othersiwse it is not meaningful for Mobile Center testing.